<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidget</class>
 <widget class="QWidget" name="SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>1800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_Background" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>1800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/backgrounds/bg_blue_day.png);
</string>
   </property>
   <widget class="QComboBox" name="comboBoxLanguage">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>400</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
border: 1px solid #D3D3D3;
border-radius: 8px;
background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(242, 242, 249, 255), stop:1 rgba(255, 255, 255, 255));
color: #333;
padding: 0px;
} 
QComboBox:on {
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #D5D5D5, stop: 1 #EEEEEE);
} 
QComboBox::drop-down {
border: 0px solid;
border-radius: 0px;
}
QComboBox::down-arrow:on {
}
</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_Settings_Icon" native="true">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>1640</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icons/settings_day.png) 0 0 0 0 stretch stretch;
</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelLanguage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Language</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
